### PR TITLE
[FIX] account_debt_report: invoice detail on debt report

### PR DESCRIPTION
**Pull request by mem-adhoc**
Ticket: 55619

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://runbot.adhoc.com.ar/ingadhoc/account-financial-tools/pull/360).

File: account_debt_report/models/res_partner.py [hunk 1]
```diff
@@ -96,12 +96,18 @@ def get_line_vals(
             detail_lines = []
             if show_invoice_detail:
                 for inv_line in record.move_id.invoice_line_ids:
+                    inv_line_name = inv_line.name
```
Comment (zaoral) [inline account_debt_report/models/res_partner.py]:
```suggestion
                    inv_line_name = inv_line.name or "Sin descripción"
```

File: account_debt_report/models/res_partner.py [hunk 2]
```diff
@@ -96,12 +96,18 @@ def get_line_vals(
...
+                    inv_line_product_uom_id_name = inv_line.product_uom_id.name
```
Comment (zaoral) [inline account_debt_report/models/res_partner.py]:
```suggestion
                    inv_line_product_uom_id_name = inv_line.product_uom_id.name or "Sin unidad de medida"
```

File: account_debt_report/models/res_partner.py [hunk 3]
```diff
@@ -96,12 +96,18 @@ def get_line_vals(
...
+                    if not inv_line_name:
+                        inv_line_name = "Sin descripción"
+                    if not inv_line_product_uom_id_name:
+                        inv_line_product_uom_id_name = "Sin unidad de medida"
```
Comment (zaoral) [inline account_debt_report/models/res_partner.py]:
```suggestion
```

**Review by zaoral — CHANGES_REQUESTED** [review]
Hi @mem-adhoc 
Thank you for the PR.
I have added some suggested changes.
Please review and let me know.

Best Regards

**Review by zaoral — APPROVED** [review]

**Comment by zaoral** [comment]
@roboadhoc r+

**Comment by fw-bot-adhoc** [comment]
This pull request has forward-port PRs awaiting action (not merged or closed): ingadhoc/account-financial-tools#361

**Comment by fw-bot-adhoc** [comment]
This pull request has forward-port PRs awaiting action (not merged or closed): ingadhoc/account-financial-tools#361

**Comment by fw-bot-adhoc** [comment]
This pull request has forward-port PRs awaiting action (not merged or closed): ingadhoc/account-financial-tools#361

**Comment by fw-bot-adhoc** [comment]
This pull request has forward-port PRs awaiting action (not merged or closed): ingadhoc/account-financial-tools#361

**Comment by fw-bot-adhoc** [comment]
This pull request has forward-port PRs awaiting action (not merged or closed): ingadhoc/account-financial-tools#361

**Comment by fw-bot-adhoc** [comment]
This pull request has forward-port PRs awaiting action (not merged or closed): ingadhoc/account-financial-tools#361

**Comment by fw-bot-adhoc** [comment]
This pull request has forward-port PRs awaiting action (not merged or closed): ingadhoc/account-financial-tools#361

**Comment by fw-bot-adhoc** [comment]
This pull request has forward-port PRs awaiting action (not merged or closed): ingadhoc/account-financial-tools#361

**Comment by fw-bot-adhoc** [comment]
This pull request has forward-port PRs awaiting action (not merged or closed): ingadhoc/account-financial-tools#361

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/account-financial-tools#374

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/account-financial-tools#374

**Pull request closed without merging**